spec/models/grda_warehouse/hud/client_spec.rb

```diff
@@ -35,11 +35,12 @@
     let!(:client) { create :grda_warehouse_hud_client }
 
     before(:all) do
-      @paper_trail_was = PaperTrail.enabled?
       PaperTrail.enabled = true
+      PaperTrail.request.enabled = true
```

Comment:
These tests started failing. The fix was to set PaperTrail.request.enabled. I'm still unsure as to the underlying cause